source/events/index.rst

```diff
@@ -1,301 +1,16 @@
-run (i.e. ``OBS_ID``). Probably the scheme to uniquely identify events at the
-DL0 level for CTA will be even more complicated, because of the much larger
-number of telescopes and events.
-
-So given that data taking and event identification is different for every IACT
-at low data levels and is already fixed for existing IACTs, we propose here
-to have an ``EVENT_ID`` that is simpler and works the same for all IACTs at
-the DL3 level.
-
-As an example: for H.E.S.S. we achive this by using an INT64 for ``EVENT_ID``
-and to store ``EVENT_ID = (BUNCH_ID_HESS << 32) || (EVENT_ID_HESS)``, i.e.
-use the upper bits to contain the low-level bunch ID and the lower bits
-to contains the low-level event ID.
-This encoding is unique and reversible, i.e. it's easy to go back to
-``BUNCH_ID_HESS`` and ``EVENT_ID_HESS`` for a given ``EVENT_ID``,
-and to low-level checks (e.g. look at the shower images for a given event
-that behaves strangely in reconstructed high-level parameters).
-
-
-.. _iact-gti:
-
-GTI extension
--------------
-
-Each event list file contains an extension to specify the good time intervals
-('GTIs'). A general description of GTIs can be found in the `OGIP GTI`_
-standard. This HDU contains two columns named START and STOP. At least one row
-is containing the start and end time of the observation must be present. The
-values are in units of seconds with respect to the reference time defined in the
-associated header (keywords MJDREFI and MJDREFF). This extension allows for a
-detailed handling of good time intervals (i.e. excluding periods with cloud
-cover or lightning during one observation). Eventually, this extension could
-disappear from the required extensions. High-level Science tools could add the
-GTIs to the files according to user parameter. See e.g. `gtmktime`_ for an
-application example from the Fermi Science Tools. The column names and FITS
-header keywords are documented in the following, respectively.
-
-GTI Column Names
-----------------
-
-* ``START`` tform: ``1D``, unit: s
-    * Start time of good time interval (given in instrument specific time reference, see below)
-* ``STOP`` tform: ``1D``, unit: s
-    * End time of good time interval (given in instrument specific time reference, see below)
-    
-GTI Header Keywords
--------------------
+This document describes the IACT DL3 event list format.
 
-* ``MJDREFI`` type: int, unit: days
-    * Integer part of instrument specific MJD time reference
-* ``MJDREFF`` type: float, unit: days
-    * Float part of instrument specific MJD time reference       
-* ``TIMEUNIT`` type: string
-    * Time unit (e.g. 's')
-* ``TIMESYS`` type: string
-    * Time system (e.g. 'TT', 'MJD', 'JD', 'TJD')
-* ``TIMEREF`` type: string
-    * Time reference frame, used for example for barycentric corrections
-      (options: 'LOCAL', 'SOLARSYSTEM', 'HELIOCENTRIC', 'GEOCENTRIC')
+Event lists are stored in FITS files that contain the following
+extensions (HDUs):
```

Comment:
You say that GTI and POINTING are part of the event list here.
For GTI that might be acceptable (but is also controversial), for POINTING it's even less clear to me if that grouping is good, or if POINTING shouldn't be in some section separate from EVENTS (what is always called TECH data in CTA).

Maybe this page would already be a good place to say that POINTING is optional (at the moment)?
Is GTI optional or required at the moment?
I guess you think it should be required and tools should get times from there by default (not the EVENTS FITS header)?
That's fine with me, because it's more flexible and not a big burden to make everyone produce GTI.